internal/session/query.go

```diff
@@ -124,6 +124,15 @@ from
 	session_connection_limit, session_connection_count;
 `
 
+	// connectConnection sets the connected time range to (now, infinity) to
+	// indicate the connection is connected.
+	connectConnection = `
+    update session_connection 
+        set connected_time_range=tstzrange(now(),'infinity') 
+    where 
+        public_id=@public_id
```

Comment:
At least for these new queries, can we format them as per the style guide, ie:

```suggestion
    update session_connection 
       set connected_time_range = tstzrange(now(), 'infinity') 
     where public_id=@public_id
```